modules/tinymce/src/core/main/ts/api/fmt/StyleFormat.ts

```diff
@@ -1,6 +1,6 @@
 import { BlockFormat, InlineFormat, SelectorFormat } from './Format';
 
-// somewhat documented at https://www.tiny.cloud/docs/configure/content-formatting/#style_formats
+// somewhat documented at https://www.tiny.cloud/docs/tinymce/6/content-formatting.html#_formats
```

Comment:
Another ID that should be fixed (i.e. not using an underscore prefixed ID). This one probably doesn't matter though given it's only a comment and not something used in the generated docs.